0800-appendix-197653-perow.Rmd

```diff
@@ -118,8 +118,8 @@ located on Highway 16, then an additional 25m downstream from the
 highway.  The entire area assessed downstream of the crossing was dry at the time of the survey 
 `r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-197653-01) - \\@ref(fig:photo-197653-02)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-197653-d01))."))`
 `r fpr_my_habitat_paragraph(loc = 'ds')` PSCIS crossing 197657 on the highway, approximately 50m
-downstream from crossing 197653 was assessed and ranked as passable as it was low gradient, fully embedded and with no outlet drop. It appeared passable to all species and life stages at the time of
-assessment (Table \@ref(tab:tab-culvert-197657)). The stream from
+downstream from crossing 197653 was reassessed and ranked as passable as it was low gradient, fully embedded and with no outlet drop. It appeared passable to all species and life stages at the time of
+assessment (Table \@ref(tab:tab-culvert-197657)). This crossing was previously assessed in 2020 by @irvine2021BulkleyRiver, but it was mistakenly input as modelled crossing ID 1800064, when it should have been 1800067. The stream from
```

Comment:
to close the issue it needs to be input into the pscis fixes repo as per the note from Simon.